get-started/document-agent's-general-description+purpose.md

```diff
@@ -0,0 +1,17 @@
+## Cryostat Agent's general description + purpose
+
+The Cryostat agent is a new component of Cryostat, an application used for retrieving data from Java applications that runs on the Java Virtual Machine (JVM). The Cryostat agent is a Java agent that acts as a plugin system for the applications running on the JVM. Prior to the agent, Cryostat used the JMX to extract data from the JVM by initiating a conversation with those JVMs. It then fetches the data from JFR and pulls it over the network back toward the Cryostat to be accessible to end users.
+
+The Agent works differently. It is responsible for fetching data from the JVM and sending it back to Cryostat. The Agent works by looking for data within itself and the application it is plugged into. The Cryostat Agent locates Cryostat applications running on the network and it is able to initiate a conversation with those applications. It is also able to communicate back to Cryostat about the Cryostat application and how to reach it. The Cryostat agent also pushes its own Java Flight Recorder (JFR) data back to Cryostat by first initiating a network connection with Cryostat, which then analyzes and saves the data to makes it accessible to end users.   
+
+The programming interface for Cryostat is typically designed to meet its specific requirements, rather than being generalized and flexible. The benefit of this is that the security considerations are easier to understand and model, but it also means Cryostat does not interoperate with other tools such as JDK Mission Control, visualvm, jconsole, and hawtio, etc. 
+
+### The Cryostat Agent's Purpose
+
+* The Cryostat agent retrieves a wide range of information from those Cryostat applications such as memory usage, CPU utilization, etc. 
+* The Cryostat analyzes these collected data to identify problems that might be affecting the application’s performance.
+* The Agent is a third-party for developers which can be installed on the target JVM program through the command-line arguments or directly attaching to the running JVM instance.
```

Comment:
"is a third-party (?)"

third-party component, perhaps? Or be more direct and say "third-party Java Instrumentation Agent"?